test/extensions/filters/http/wasm/test_data/BUILD

```diff
@@ -5,15 +5,68 @@ load(
     "envoy_cc_library",
     "envoy_package",
 )
-load("//bazel/wasm:wasm.bzl", "wasm_cc_binary")
+load("//bazel/wasm:wasm.bzl", "wasm_cc_binary", "wasm_rust_binary")
 
 envoy_package()
 
-filegroup(
-    name = "modules",
-    srcs = glob([
-        "*.wasm",
-    ]),
+wasm_rust_binary(
+    name = "async_call_rust.wasm",
+    srcs = ["async_call_rust/src/lib.rs"],
+    rustc_flags = ["-Clink-arg=-zstack-size=32768"],
+    deps = [
+        "//bazel/external/cargo:log",
+        "//bazel/external/cargo:proxy_wasm",
+    ],
+)
+
+wasm_rust_binary(
+    name = "body_rust.wasm",
+    srcs = ["body_rust/src/lib.rs"],
+    rustc_flags = ["-Clink-arg=-zstack-size=32768"],
+    deps = [
+        "//bazel/external/cargo:log",
+        "//bazel/external/cargo:proxy_wasm",
+    ],
+)
+
+wasm_rust_binary(
+    name = "headers_rust.wasm",
+    srcs = ["headers_rust/src/lib.rs"],
+    rustc_flags = ["-Clink-arg=-zstack-size=32768"],
+    deps = [
+        "//bazel/external/cargo:log",
+        "//bazel/external/cargo:proxy_wasm",
+    ],
+)
+
+wasm_rust_binary(
+    name = "metadata_rust.wasm",
+    srcs = ["metadata_rust/src/lib.rs"],
+    rustc_flags = ["-Clink-arg=-zstack-size=32768"],
+    deps = [
+        "//bazel/external/cargo:log",
+        "//bazel/external/cargo:proxy_wasm",
+    ],
+)
+
+wasm_rust_binary(
+    name = "shared_data_rust.wasm",
+    srcs = ["shared_data_rust/src/lib.rs"],
+    rustc_flags = ["-Clink-arg=-zstack-size=32768"],
+    deps = [
+        "//bazel/external/cargo:log",
+        "//bazel/external/cargo:proxy_wasm",
+    ],
+)
+
+wasm_rust_binary(
+    name = "shared_queue_rust.wasm",
+    srcs = ["shared_queue_rust/src/lib.rs"],
+    rustc_flags = ["-Clink-arg=-zstack-size=32768"],
```

Comment:
Why do we have this stack size boiler plate?

Reply:
There is one test that verifies linear memory size for `test_rust.rs`, but it can be removed from other tests... and `test_rust.rs` can be probably completely removed now that we have tests written using Proxy-Wasm Rust SDK.

Reply:
Just deleted unused stack size specification right now. I'm not sure if the `test_rust.rs` is required so missing it.